galsim/fitswcs.py

```diff
@@ -1296,14 +1338,39 @@ def _radec(self, x, y, color=None):
 
     def _invert_pv(self, u, v):
         # Do this in C++ layer for speed.
-        with convert_cpp_errors():
-            return _galsim.InvertPV(u, v, self.pv.ctypes.data)
+        try:
+            len(u)
+        except TypeError:
+            with convert_cpp_errors():
+                return _galsim.InvertPV(u, v, self.pv.ctypes.data)
+        else:
+            invpvu = np.empty(len(u))
+            invpvv = np.empty(len(u))
+            for i in range(len(u)):
+                with convert_cpp_errors():
+                    uu, vv = _galsim.InvertPV(u[i], v[i], self.pv.ctypes.data)
```

Comment:
The `InvertPV` and `InvertAB` functions are already doing a non-linear solution to a polynomial equation, so I'd be very surprised if leaving the outer loop in python is a slow down.  If someone can demonstrate otherwise, we can adjust to let these functions take numpy arrays, but it didn't seem worth worrying about to me.